deployments/k8s/helm/signalfx-agent/values.yaml

```diff
@@ -2,9 +2,14 @@
 # docker image tag if not overridden with imageTag
 agentVersion: 4.14.1
 
-# The access token for SignalFx. (REQUIRED)
+# The access token for SignalFx. (REQUIRED if signalFxAccessTokenExistingSecret not set)
 signalFxAccessToken: ""
 
+# Use an existing secret for the SFX token (if set does not create a secret)
+# See templates/secrets.yaml for the format of the secret to use
+# If set, signalFxAccessToken is not required
+signalFxAccessTokenExistingSecret: ""
```

Comment:
Would prefer this called `signalFxAccessTokenSecretName` instead so that it is clear that is is the _name_ of the secret resource.